samples/FXL_I2C/prj.conf

```diff
@@ -0,0 +1,7 @@
+CONFIG_LOG=y
```

Comment:
Please check which ones of these are already included in the board/SoC Kconfig, and separate them like this (or similar):

`### required for driver but selected by board/SoC already ###`
`CONFIG_GPIO=y`
`CONFIG_SERIAL=y`

`### required for driver and not selected by board/SoC ###`
`CONFIG_MODEM=y`
`CONFIG_MODEM_UBLOX_SARA_N310=y`